src/client/entity-editor/relationship-editor/relationship-editor.tsx

```diff
@@ -405,17 +406,28 @@ class RelationshipModal
 			// Name of the attribute type belonging to the relationship type. EX: ['position', 'number]
 			attributes = attributeTypes.map(attribute => attribute.name);
 		}
+		const colourStyles = {
+			option: (styles) => ({
+				...styles,
+				backgroundColor: 'transparent',
+				color: 'inherit'
+			  })
+		  };
```

Comment:
This is not needed since we are using classes now to style components, i used it as promoted by [react-select](https://react-select.com/upgrade-to-v2#new-styles-api)  but later realized classes would be more convenient here.